packages/app/src/providers/api.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/member-ordering */
```

Comment:
This rule forces me to have non-abstract methods before abstract methods. `getExportURL` and `getSearchablePaths` can't be abstract because they're optional (i.e. if I make them abstract, I have to implement them in all providers) but I still prefer them to appear after the main abstract methods like `getValue`).